packages/frontend/src/components/common/ErrorState/index.tsx

```diff
@@ -56,7 +49,12 @@ const ErrorState: FC<{
         }
     }, [error]);
 
-    return <StyledSuboptimalState $hasMarginTop={hasMarginTop} {...props} />;
+    return (
+        <SuboptimalState
+            style={{ marginTop: hasMarginTop ? '20px' : undefined }}
```

Comment:
underneath `SuboptimalState` is Mantine's Stack but it does not forward props correctly.

ideally SuboptimalState's state should be 

`interface Props extends StackProps { ... }`

and rest + sx spread


```
            {...rest}
            sx={{
              ...rest?.sx
            }}

```